### PR TITLE
fix(appeals): updated folder names for new notify rows

### PR DESCRIPTION
**Pull request by BiggsKC**
## Describe your changes

Document types fix:
- siteNotice -> whoNotifiedSiteNotice
- lettersToNeighbours -> whoNotifiedLetterToNeighbours
- pressAdvert -> whoNotifiedPressAdvert

## Issue ticket number and link

[A2-315 Creation of three new rows/folders for HAS LPAQ](https://pins-ds.atlassian.net.mcas.ms/browse/A2-315)

## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

**Review by james-hill-kc — DISMISSED** [review]
LGTM!

File: appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/lpa-questionnaire.controller.js [hunk 1]
```diff
@@ -253,13 +253,13 @@ export const getAddDocuments = async (request, response) => {
 	let uploadPageHeadingText = '';
 
 	switch (documentType) {
-		case 'siteNotice':
+		case 'whoNotifiedSiteNotice':
```
Comment (fabfab-kc) [inline appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/lpa-questionnaire.controller.js:256]: These could be changed to the constants, i.e. `APPEAL_DOCUMENT_TYPE.WHO_NOTIFIED_SITE_NOTICE`...

Comment (BiggsKC) [reply appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/lpa-questionnaire.controller.js]: sorted. also for the `notificationDocumentTypes` array below

**Review by fabfab-kc — APPROVED** [review]
LGTM!

**Pull request merged**